velox/common/base/SpillStats.h

```diff
@@ -140,4 +140,9 @@ void updateGlobalMaxSpillLevelExceededCount(
 
 /// Gets the cumulative global spill stats.
 SpillStats globalSpillStats();
+
+/// Formatter for fmt.
+auto format_as(SpillStats f) {
```

Comment:
f -> stats

auto -> std::string

Using 'auto' for return types makes it very hard to read the code.